src/main/webapp/manage-availability.js

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function getAvailabilityManage() {
+    var queryString = new Array();
+    window.onload = readTutorID(queryString, window);
+    const tutorID = queryString["tutorID"];
+
+    document.getElementById("tutorEmail").value = tutorID;
+
+    const params = new URLSearchParams();
+    params.append('tutorID', tutorID);
+    fetch('/availability', {method: 'POST', body: params}).then(response => response.json()).then((timeslots) => {
```

Comment:
I think this would make more sense as a GET request.

Reply:
The thing is that GET requests don't take parameters and I need to pass the tutorID as a parameter in order to load that tutor's availability.

Reply:
You can do `/availability?<parameter name>=<value>` to pass parameters in a GET request. Then in the servlet, you can do `request.getParameter(<parameter name>)`.

Reply:
Oh this would be great!! Thank you so mcuh

Reply:
I realized that the addavailability servlet is actually writing to the database so I think it should be doPost. However, my availability servlet is using a doPost function when it should be in fact just using a doGet because in that servlet it just reads the available time slots. I will create a new PR that addresses this issue if that's okay just so that this one doesn't get too complex.